scripts/travis.sh

```diff
@@ -15,12 +15,14 @@ cd "$(dirname "$0")/.."
 case "$1" in
 
     "build")
-        bazel build --show_progress_rate_limit=2 //...
+        bazel build --show_progress_rate_limit=2 //hocon-compiler
```

Comment:
we can probably remove this now; I'll do that on the next PR so builds don't have to run again